app/src/main/java/com/ndt/themoviedb/ui/base/BaseFragment.kt

```diff
@@ -0,0 +1,37 @@
+package com.ndt.themoviedb.ui.base
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.viewbinding.ViewBinding
+
+abstract class BaseFragment<VBinding : ViewBinding>(private val bindingLayoutInflater: (LayoutInflater) -> VBinding) :
+    Fragment() {
+
+    private var binding: ViewBinding? = null
+    protected val mViewBinding: ViewBinding
```

Comment:
```suggestion
    private var _binding: ViewBinding? = null
    protected val binding: ViewBinding
```
mình dùng _ để đại diện cho biến private